projects/demo/src/modules/markup/breakpoints/examples/1/index.less

```diff
@@ -26,7 +26,7 @@
 @breakpoints: tui-mobile, tui-mobile-min, tui-mobile-interval, tui-tablet, tui-tablet-min, tui-tablet-interval,
     tui-desktop, tui-desktop-min, tui-desktop-interval, tui-desktop-lg-min;
 
-each(@breakpoints, {
+.tui-for(@breakpoints, {
```

Comment:
I am not sure that we need to expand public API only for Stackblitz.

Maybe we can change only this file (replace this modern `each` by legacy-way loop)?
Or should we make "magic" inside this file ?
https://github.com/Tinkoff/taiga-ui/blob/7f1fdf880f8268d0cb207a346468c81b31d4cddd/projects/demo/src/modules/app/stackblitz/utils.ts#L6